locale/translations/en_US.json

```diff
@@ -123,6 +123,7 @@
     "page.keyboard_shortcuts.download_content": "Download original content",
     "page.keyboard_shortcuts.toggle_bookmark_status": "Toggle bookmark",
     "page.keyboard_shortcuts.save_article": "Save article",
+    "page.keyboard_shortcuts.remove_feed": "Remove this feed",
```

Comment:
To make the unit tests pass, you have to add this key in other language files. You can reuse existing translation for the value.

Reply:
Good idea! I followed your advice. *But* I could not make neither the unit tests (`make test`) nor the integration tests (`make integration-test`) fail, if I removed a translation on purpose. Just a heads up. I probably did something wrong.. :)